code-challenges.js

```diff
@@ -10,7 +10,48 @@ var people = [
 ]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is a president of the galaxy." "Arthus Dent is a radio employee."
 
+const sent = (obj) => {
 
+  //capture the length of the object being passed down in order to use in future loops
+  let length = obj.length
+
+  //capture all the "names" in the object
+  let arrNames = obj.map(value => value.name)
+
+  //seperate the first and last names 
+  let arrCap = arrNames.map(value => value.split(" "))
+
+  //once the first and last names are seperated we can now go in and captalize the first letter.
+  let name1 = arrCap.map(value => value[0].charAt(0).toUpperCase() + value[0].slice(1))
```

Comment:
duplication of code here suggests there can be a way to iterate or do mapping first then pull values into individual variables. Basically here, `arrCap` is being looped on twice with the same end result.